wapitiCore/attack/attack.py

```diff
@@ -558,7 +558,7 @@ def read_payloads(self, filename):
         """returns a array"""
         lines = []
         try:
-            with open(filename, errors="ignore") as file:
+            with open(filename, errors="ignore", encoding='utf-8') as file:
```

Comment:
Wow! I was pretty sure the default value on `open()` for encoding was `utf-8`, thank you for fixing this.